### PR TITLE
add estimator demo

### PR DESCRIPTION
**Pull request by tracyxzh01**
# Description

Brief Description of the PR:

Fixes # (issue)

## Type of change

- [ ] Bug fix
- [ ] New Tutorial
- [ ] Updated or additional documentation
- [ ] Additional Testing
- [ ] New Feature

# Checklist:

- [ ] I've properly [formatted my code according to the guidelines](https://github.com/tensorflow/recommenders-addons/blob/master/CONTRIBUTING.md#coding-style)
    - [ ] By running Black + Flake8
    - [ ] By running pre-commit hooks
- [ ] This PR addresses an already submitted issue for TensorFlow Recommenders Addons
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] This PR contains modifications to C++ custom-ops

# How Has This Been Tested?

If you're adding a bugfix or new feature please describe the tests that you ran to verify your changes:
*

### CODE REVIEW

File: demo/movielens-100k-estimator/README.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# A distributed training demo for `tfra.dynamic_embedding`:
+
+- dataset: [movielen/100k-ratings](https://www.tensorflow.org/datasets/catalog/movielens#movielens100k-ratings)
+- model: DNN
+- Running mode: Graph mode by using MonitoredTrainingSession
```
Comment (rhdong) [inline demo/movielens-100k-estimator/README.md:5]: change to `estimator APIs`

File: demo/movielens-100k-estimator/movielens-100k-estimator.py [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+def serving_input_receiver_dense_fn():
+  input_spec = {
+      "movie_id": tf.constant([1], tf.int64),
+      "user_id": tf.constant([1], tf.int64),
+      "user_rating": tf.constant([1.0], tf.float32)
+  }
+  return tf.estimator.export.build_raw_serving_input_receiver_fn(input_spec)
+
+
+def export(model_dir, export_dir):
```
Comment (rhdong) [inline demo/movielens-100k-estimator/movielens-100k-estimator.py:148]: `export_for_serving` maybe better?

File: demo/movielens-100k-estimator/README.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
+# A distributed training demo for `tfra.dynamic_embedding`:
+
+- dataset: [movielen/100k-ratings](https://www.tensorflow.org/datasets/catalog/movielens#movielens100k-ratings)
+- model: DNN
+- Running mode: Graph mode by using estimator APIs
+
+## start train:
+By default, this shell will start a train task with 2 PS and 2 workers on local machine.
```
Comment (Mr-Nineteen) [inline demo/movielens-100k-estimator/README.md:8]: Is it better to add a chief role?

File: demo/movielens-100k-estimator/start.sh [hunk 4]
```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+rm -rf ./ckpt
+rm -rf ./export_dir
+sh stop.sh
+sleep 1
+python movielens-100k-estimator.py --ps_list="localhost:2220,localhost:2221" --worker_list="localhost:2230,localhost:2231" --task_mode="ps" --task_id=0 &
```
Comment (Mr-Nineteen) [inline demo/movielens-100k-estimator/start.sh:6]: Is it better to use TF_CONFIG?

File: demo/movielens-100k-estimator/movielens-100k-estimator.py [hunk 5]
```diff
@@ -0,0 +1,169 @@
...
+                                        save_summary_steps=100,
+                                        save_checkpoints_steps=100,
+                                        save_checkpoints_secs=None)
+
+  estimator = tf.estimator.Estimator(model_fn=model_fn,
+                                     model_dir=model_dir,
+                                     params=None,
+                                     config=model_config)
+
+  estimator.export_saved_model(export_dir, serving_input_receiver_dense_fn())
```
Comment (funsimple) [inline demo/movielens-100k-estimator/movielens-100k-estimator.py:159]: Is saving 4 models in line with expectations, when I run `start.sh`?

Comment (tracyxzh001) [reply demo/movielens-100k-estimator/movielens-100k-estimator.py:159]: bugfix pr https://github.com/tensorflow/recommenders-addons/pull/70/files

**Comment by tracyxzh01** [comment]
new pr  https://github.com/tensorflow/recommenders-addons/pull/70/files

**Pull request closed without merging**